internal/flags/activate.go

```diff
@@ -16,6 +16,7 @@ func (f *Flags) handleActivateCommand() utils.ReturnCode {
 	f.amtActivateCommand.BoolVar(&f.Local, "local", false, "activate amt locally")
 	f.amtActivateCommand.BoolVar(&f.UseCCM, "ccm", false, "activate in client control mode (CCM)")
 	f.amtActivateCommand.BoolVar(&f.UseACM, "acm", false, "activate in admin control mode (ACM)")
+	f.amtActivateCommand.StringVar(&f.UUID, "uuid", "", "uuid override (non-CIRA workflow)")
```

Comment:
"uuid override (non-CIRA workflow)", i might suggest something like "override AMT device uuid for use with non-CIRA workflow". As it stands, it doesnt tell me what i'm overriding.